include/gatekeeper_l2.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Gatekeeper - DoS protection system.
+ * Copyright (C) 2016 Digirati LTDA.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _GATEKEEPER_L2_H_
+#define _GATEKEEPER_L2_H_
+
+#include "gatekeeper_net.h"
+
```

Comment:
We should have an include for `struct rte_mbuf`, `struct ether_hdr`, `struct vlan_hdr`, `RTE_PTYPE_L2_ETHER_VLAN`, and `RTE_PTYPE_UNKNOWN`.